code/graphics/2d.cpp

```diff
@@ -162,17 +165,18 @@ static bool mode_change_func(os::ViewportState state, bool initial)
 	return true;
 }
 
-static auto WindowModeOption __UNUSED = options::OptionBuilder<os::ViewportState>("Graphics.WindowMode", "Window Mode",
-																		 "Controls how the game window is created.")
-								   .category("Graphics")
-								   .level(options::ExpertLevel::Beginner)
-								   .values({{os::ViewportState::Fullscreen, "Fullscreen"},
-											{os::ViewportState::Borderless, "Borderless"},
-											{os::ViewportState::Windowed, "Windowed"}})
-								   .importance(98)
-								   .default_val(os::ViewportState::Fullscreen)
-								   .change_listener(mode_change_func)
-								   .finish();
+static auto WindowModeOption __UNUSED = options::OptionBuilder<os::ViewportState>("Graphics.WindowMode",
+                     std::pair<const char*, int>{"Window Mode", 1772},
+                     std::pair<const char*, int>{"Controls how the game window is created", 1773})
+                     .category("Graphics")
+                     .level(options::ExpertLevel::Beginner)
+                     .values({{os::ViewportState::Fullscreen, {"Fullscreen", 1674}},
```

Comment:
Sorry, it looks like I missed this earlier, but isn't ID 1674 already reserved for "Nothing"?